### PR TITLE
Unfold notification rollups

### LINKED ISSUE #1423: Retags isnt working for me!
Hi there, the activity page is giving me a little trouble. I had the 'Retags' extension, so I used to be able to see the individual retags every time someone reblogged a post from me. but now, I see this instead:
image

The tags at the bottom are my own tags. I cant see other people's tags any more. 

image

This started about a month ago when the activity page got an update and started to bunch reblogs together (e.g. "person and 16 others reblogged your post"). 

When I toggle the 'show only retags / responses' switch off, I get something like this:
image

I'm looking for a solution so I can see the tags people use when they reblog my art again. I asked on the discord but I think not everyone has this problem.

If it's possible to find a fix I would really appreciate that. 

EDIT: Retags works fine when I see the notifications on the dashboard (the temporary little ones between posts) 
image

### PR DESCRIPTION
**Pull request by hobinjk**
Still very WIP but it kinda does some things

notes

To-do:
- [x] Format reblogs without text properly
- [x] Fastforward notes API to actual note using before_timestamp and url checking
- [x] Fix Activity+ condensing
---
resolves #1423

### CODE REVIEW

**Comment by hobinjk** [comment]
There's also a bug in retags where it should be processing is_reblog_naked but isn't

**Comment by hobinjk** [comment]
better notes

**Comment by hobinjk** [comment]
Review ready and unrolling notes!

**Comment by hobinjk** [comment]
The main bad thing about this is that it has no localization in its giant blob of html. It actually only does ~6 API requests in the worst case per-notification.

Since I think I'm the only dev with this Tumblr "feature" enabled I think it makes sense for the review to be just reading the code. Let me know if you want screenshots or other verification

File: Extensions/activity_plus.js [hunk 1]
```diff
@@ -264,6 +416,7 @@ XKit.extensions.activity_plus = new Object({
 
 		if ((XKit.extensions.activity_plus.condensed_count - 1) <= 1) { return; }
 
+
```
Comment (AprilSylph) [inline Extensions/activity_plus.js]: double newline x2 combo

File: Extensions/activity_plus.js [hunk 2]
```diff
@@ -234,6 +377,7 @@ XKit.extensions.activity_plus = new Object({
 						$(this).attr('xkit-activity-plus-condense-id', XKit.extensions.activity_plus.condensed_id);
 						XKit.extensions.activity_plus.condensed_count++;
 
+
```
Comment (AprilSylph) [inline Extensions/activity_plus.js]: double newline?

**Comment by hobinjk** [comment]
When I press enter I think to myself "yes"
When I press delete I think to myself "no"

**Pull request merged**